package.json

```diff
@@ -8,6 +8,19 @@
     "icons"
   ],
   "main": "index.js",
```

Comment:
Did you try to add this one as well?

```suggestion
  "type": "commonjs",
  "main": "index.js",
```

Reply:
This is the default, so it shouldn't be necessary, though we can add it if you want to be explicit about it.

Reply:
True. I don't think it hurts to be explicit about it though :thinking: